src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -619,10 +619,10 @@ public string GetGithubRepoName(string repoUrl)
             }
 
             StringBuilder builder = new StringBuilder(repoName);
+            int index = repoName.IndexOf('-');
 
-            if (repoName.Contains("-"))
+            if (index > -1)
```

Comment:
```suggestion
            if (index > 0)
```

Otherwise this is still broken